src/app/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,46 @@
+import {Component, signal} from '@angular/core';
+import {Router, RouterLink} from "@angular/router";
+import {InfoCardComponent} from "./info-card/info-card.component";
+import {NgClass} from "@angular/common";
+import {Books} from "../../mocks/books";
+import {FooterComponent} from "../footer/footer.component";
+import {IBook} from "../../models/book";
+import {IComponentInformation} from "../../models/component-information";
+
+@Component({
+  selector: 'app-landing-page',
+  standalone: true,
+  imports: [
+    RouterLink,
+    InfoCardComponent,
+    NgClass,
+    FooterComponent
+  ],
+  templateUrl: './landing-page.component.html',
+  styleUrl: './landing-page.component.scss'
+})
+export class LandingPageComponent {
+  books: IBook[] = Books;
+
+
+  componentInformation: IComponentInformation[] = [
```

Comment:
nazwa `IComponentInformation` bez kontekstu jest mało zrozumiała

Domyslam sie ze to na potrzeby inputów w komponencie jest uzywane, jednak jakiego komponentu i jakie informacje? 

Lepiej użyć bardziej domenowych nazw. Czeba zadać sobie pytanie gdzie to jest wrzucone i czego konkretnie dotyczą te dane i tak właśnie nazywać zmienne i typy.